listenbrainz/webserver/__init__.py

```diff
@@ -102,6 +103,9 @@ def gen_app(config_path=None, debug=None):
     # Timescale connection
     create_timescale(app)
 
+    # Metrics
+    metrics.init('listenbrainz')
```

Comment:
Currently this is what we set `key` to, it'll the same for all containers. If we want this to be different for each service, we should init it here to the env variable CONTAINER_NAME, but we should also include an indicator that it's from listenbrainz: something like `init('listenbrainz', os.environ['CONTAINER_NAME'])` This would internally store that the second argument is a metric of listenbrainz, and the `/internal/stats` endpoint can return a list of all keys under the listenbrainz namespace.

Do we also want to add the hostname to this init?

Reply:
The host key should be set in each metric as per our discussion in IRC today. If this is the place to add it, then yes.